setup/features/notifications/index.md

```diff
@@ -102,26 +102,13 @@ Note: your users will need to invite the hipchat bot to private rooms that want
 
 ## Slack
 
-For slack, you will need to create a custom bot user (https://api.slack.com/bot-users#how_do_i_create_custom_bot_users_for_my_team), then get the access token associated with the new bot user. 
+For slack, you need to [create a custom bot user](https://api.slack.com/bot-users#how_do_i_create_custom_bot_users_for_my_team), then get the access token associated with that new bot user. Then...
 
-in echo.yml
-```
-slack:
-  enabled: true
-  token: <API token for bot>
-```
+`hal config notification slack enable`
 
-in settings.js (deck)
-```
-window.spinnakerSettings = {
-// ...
-  notifications: {
-    slack: {
-      enabled: true,
-      botName: '<username of bot>'
-    },
-// ...
-```
+and
+
+`hal config notification slack edit --bot-name $SPINNAKER_BOT --token $TOKEN_FROM_SLACK`
```

Comment:
I would format this in one large block with 

\`\`\`bash
hal config notification slack enable
hal config notification slack edit --bot-name $SPINNAKER_BOT --token $TOKEN_FROM_SLACK
\`\`\`

Reply:
This adds bash syntax highlighting & groups the commands together